readthedocs/urls.py

```diff
@@ -34,6 +34,8 @@
     url(r'^$', HomepageView.as_view(), name='homepage'),
     url(r'^support/', SupportView.as_view(), name='support'),
     url(r'^security/', TemplateView.as_view(template_name='security.html')),
+    url(r'^.well-known/security.txt',
```

Comment:
Do we need this? Who uses it? How?

(I read the RFC at https://tools.ietf.org/html/rfc5785 but I don't understand the use case)

Reply:
I should have put more details. A `security.txt` file is not yet a standard but may be. It has been submitted to become an RFC. The goal of it is that it is a standard place where a security researcher can find the right place to disclose an issue.

https://securitytxt.org/